pilot/pkg/networking/plugin/authn/authentication_test.go

```diff
@@ -539,6 +540,84 @@ func TestBuildJwtFilter(t *testing.T) {
 	}
 }
 
+func TestBuildLocalJwtFilter(t *testing.T) {
```

Comment:
TestBuildLocalJwtFilter -> TestJwtFilterWithInlinePublicKey

Reply:
Actually, have TestConvertPolicyToJwtConfigWithInlineKey instead (i.e test the ConvertPolicyToJwt directly). BuildJwtFilter is v2 only function, so it can be test with `enableLocalJwks` always on.

Reply:
done